src/main/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/service/PrisonApiService.kt

```diff
@@ -56,7 +56,7 @@ class PrisonApiService(
       }
   }
 
-  suspend fun findPrisonersAtLocation(prisonId: String, locationId: String): Flow<PrisonerAtLocation> =
+  suspend fun findPrisonersAtLocation(locationId: String): Flow<PrisonerAtLocation> =
```

Comment:
`prisonId` was not used in the method, the Prison API endpoint only uses the `locationId` anyway.